src/nibpkg/compose.nim

```diff
@@ -18,11 +18,14 @@ proc retrieve_flanking_sequences_from_fai*(fastaIdx: Fai, chrom: string,
   result.left = fastaIdx.get(chrom, max(0, start_pos - flank), start_pos)
   result.right = fastaIdx.get(chrom, end_pos, end_pos + flank)
 
-proc kmerize(s: string, k: int = 25): seq[seed_t] =
-  return Dna(s).dna_to_kmers(k).seeds
+proc kmerize(s: string, k: int = 25, spaced_seeds : bool = false, space: int = 50): seq[seed_t] =
```

Comment:
Simpler is to let `space == 0` be the default, and when you pass `50` then use spaced seeds.

But I might have a slightly better idea. If so, I'll push something in a few minutes ...

Reply:
Ok, I've pushed onto your branch, for that minor change plus actually using spaced-seeds for the initial "alt seqs" too. That might fix the problem. (I don't have time to run locally today.) Take a look.